doctr/transforms/functional/tensorflow.py

```diff
@@ -60,17 +62,16 @@ def rotate(
 
     # Get absolute coords
     _boxes = deepcopy(boxes)
-    if boxes.dtype != int:
+    if np.max(_boxes) < 2:
```

Comment:
same here, I don't understand the max part?

Reply:
same, check for relative coords